doc/radosgw/cloud-transition.rst

```diff
@@ -288,50 +299,63 @@ For example,
             }
         }
     ]
+```
 
-Object modification & Limitations
+Object Modification and Limitations
 ----------------------------------
 
-The cloud storage class once configured can then be used like any other storage class in the bucket lifecycle rules. For example,
-
-::
+The cloud storage class once configured can then be used like any other storage
```

Comment:
s/class once configured/class, once configured,/
s/can then be used/can be used/
s/in bucket lifecycle (LC) rules/when defining bucket lifecycle (LC) rules/